README.md

```diff
@@ -162,24 +162,45 @@ const render = () => html`
 `;
 ```
 
-### Function Values / Directives
+### Directives
 
-A function valued expression can be used for error handling and stateful rendering.
+Directives Functions are functions that can extend lit-html by directly interacting with the Part API.
```

Comment:
"Directives are functions"... seems like "Directive Function" isn't helping but if you insist on this name then reword to avoid "Functions are functions"